content/guides/user-guides/csi/ceph csi/_index.md

```diff
@@ -0,0 +1,364 @@
+---
+title: CEPH CSI User Guide description: "A user guide for integrating SODA with CEPH CSI Driver via unified Plugin and
+hotpot project"
+weight: 30 disableToc: false tags: ["user guide", "csi", "kubernetes csi", "plug-n-play", "unified plugin"]
+---
+
+# SODA Integration with Vendor CSI Drivers via SODA CSI Plug-N-Play
+
+SODA CSI Plug-N-Play is a mechanism by which users can use the heterogeneous storage vendors in a unified way, Users
+need to define the requirements while creating SODA Profile, Users need to use this profile ID while creating the PVC's
+in the Kubernetes, the soda-csi-provisioner will automatically select the vendor csi-driver and help in provisioning the
+Volumes for Pods.
+
+## Prerequisite
+
+- An installation of Kubernetes (V1.17+)
+- SODA installation (you can refer to quick start guide
+  over [here](https://docs.sodafoundation.io/soda-gettingstarted/installation-using-ansible/)), you need to change the
+  ports of etcd in osdb.yaml while installing SODA on an existing K8s environment as the ports cause conflict with k8s
```

Comment:
I think it should be osdb.yaml -> osdsdb.yml.
Please check.

Reply:
yes @anvithks you are right, I have updated it